openlibrary/i18n/es/messages.po

```diff
@@ -1204,12 +1093,8 @@ msgid "Leave the Waiting List"
 msgstr "Abandonar la lista de espera"
 
 #: borrow.html:159
-msgid ""
-"Are you sure you want to leave the waiting list "
-"of<br/><strong>TITLE</strong>?"
-msgstr ""
-"¿Estás seguro de que quieres abandonar la lista de espera de "
-"<br/><strong>TITLE</strong>?"
+msgid "Are you sure you want to leave the waiting list of<br/><strong>TITLE</strong>?"
+msgstr "¿Estás seguro de que quieres abandonar la lista de espera de <br/><strong>TITLE</strong>?"
```

Comment:
Reminder to @tfmorris that the TITLE notation is confusing and should be more obviously a placeholder.